Files/Helpers/ExternalResourcesHelper.cs

```diff
@@ -33,6 +33,17 @@ public async Task LoadSelectedSkin()
                 OptionalPackageSkinFolder = package.InstalledLocation;
             }
 
+            // ToDo this is for backwards compatability, remove after a couple updates
+            var themeFolder = await StorageFolder.GetFolderFromPathAsync(ApplicationData.Current.LocalFolder.Path + "\\Themes");
```

Comment:
This could throw a ``FileNotFound`` exception if the user just installed the app and doesn't have the Themes folder, I would suggest moving it to the try catch as well.